siliconcompiler/tools/yosys/syn_fpga.tcl

```diff
@@ -65,8 +185,14 @@ if {[string match {ice*} $sc_partname]} {
     yosys flatten
     yosys clean
 
+    if { $sc_syn_flop_library ne "None" } {
+        yosys techmap -map $sc_syn_flop_library
+    }
```

Comment:
need post processing call to techmap and opt to cleanup the design afterwards like the ASIC flow